tests/Metadata/Resource/Factory/FileConfigurationMetadataTest.php

```diff
@@ -11,6 +11,7 @@
 
```

Comment:
Should I split metadata / resourcenamecollection too?

I think it's fine like this but may be cleaner splitted ofc.